metrics/density/memory_usage_inside_container.sh

```diff
@@ -15,7 +15,9 @@ SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../lib/common.bash"
 
 TEST_NAME="memory footprint inside container"
-IMAGE="alpine:3.7"
+VERSIONS_FILE="${SCRIPT_PATH}/../../versions.yaml"
+ALPINE_VERSION=$("${GOPATH}/bin/yq" read "$VERSIONS_FILE" "docker_images.alpine.version")
+IMAGE="alpine:$ALPINE_VERSION"
```

Comment:
I don't know if we can extract this 'alpine' word from from the yaml as well, or use the `alpine.url` - but even then it would feel a bit self referential - if we did not know we were using `alpine` here, we would not know to look up the `alpine` section in the yaml. Only way out I can think of would be to make the yaml section named after the test (that is, configure all the information about which container this test uses inside the yaml). That feels one step too far for me right now - so ignore this ramble - we'll live with the wiring of `alpine` here I think.